Doc/library/time.rst

```diff
@@ -563,6 +563,9 @@ Functions
       When used with the :func:`strptime` function, ``%U`` and ``%W`` are only used in
       calculations when the day of the week and the year are specified.
 
+   (5)
+      The :func:`strptime` function does not accept non-ASCII digits for numeric values.
```

Comment:
Should we mention the "non-locale-specific numeric format codes" here or, since it's not officially supported, we can be a bit lazy?